doc/doxygen/src/Glossary.dox

```diff
@@ -232,7 +235,7 @@
  *
  * @section ManagerInterface
  *
- * The @ref openassetio.managerApi.ManagerInterface.ManagerInterface
+ * The @fqref{managerApi.ManagerInterface}.ManagerInterface
```

Comment:
[fix] This links to the `ManagerInterface` constructor, probably not what was intended.

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1058/commits/948efe5518f4da207ec3438f17531c456b524965